bin/run-canonical-extensions-tests.sh

```
@@ -0,0 +1,106 @@
+	    fi
+	done
+
+	echo ''
+done
+
+echo "Waiting for completion (${#running[@]} run(s), 1 min check interval, takes at least 40 min):" && echo -n '    '
+result=()
+while [ ${#running[@]} -gt 0 ]; do
+	echo -n '.' && sleep 1m
+	temp=()
+	for entry in ${running[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)